end_to_end_testing/setup_helm3.sh

```diff
@@ -0,0 +1,17 @@
+source "$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/setup_common.sh
+
+echo "Installing Helm"
+curl -sL https://get.helm.sh/helm-v3.0.1-linux-amd64.tar.gz | tar xzv linux-amd64/helm
+sudo mv linux-amd64/helm /usr/local/bin/helm
+rm -rf linux-amd64
+helm version
+
+kubectl create namespace infra
```

Comment:
Should we mark this as a hack or a TODO since full Helm3 support for reckoner will probably be creating namespaces for us?